scipy/sparse/_data.py

```diff
@@ -210,7 +210,20 @@ def _min_or_max(self, axis, out, min_or_max, explicit):
         if out is not None:
             raise ValueError("Sparse arrays do not support an 'out' parameter.")
 
+        if axis == ():
+            return self.copy()
```

Comment:
Let's raise an error in this case, as it's a no-op and unlikely to be what the user expected.